readme.md

```diff
@@ -54,6 +54,34 @@ Filename. Mutually exclusive with the `extension` option.
 
 Get a temporary directory path. The directory is created for you.
 
+### tempy.write(fileContent, options?)
+
+Write a string/buffer/stream to a random temp file.
```

Comment:
Don't forget to keep the readme in sync with the TS file.

Reply:
Fixed!